tutorials/kubernetes.md

```diff
@@ -1,5 +1,7 @@
 # Running CRI-O on kubernetes cluster
 
+This tutorial assumes you've already installed and setup CRI-O. If you have not, start [here](setup.md).
```

Comment:
Add a link to the start title in that page: setup.md to  setup.md#starting-cri-o